data.sql

```diff
@@ -70,4 +70,62 @@ WHERE animals.name = 'Charmander' AND owners.full_name = 'Melody Pond'
 UPDATE animals
 SET owner_id = owners.id FROM owners
 WHERE animals.name = 'Angemon' AND owners.full_name = 'Dean Winchester'
-  OR animals.name = 'Boarmon' AND owners.full_name = 'Dean Winchester';
\ No newline at end of file
+  OR animals.name = 'Boarmon' AND owners.full_name = 'Dean Winchester';
+
+  INSERT INTO vets (name, age, date_of_graduation) VALUES
+  ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
+  ('Jack Harkness', 38, '2008-06-08');
```

Comment:
- [ ] Let us modify our insertion using this way, to make our queries more readable:

  `INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('William Tatcher', 45, '2000-04-23');`

   `INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Maisy Smith', 26, '2019-01-17');`

   `INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Stephanie Mendez', 64, '2019-05-17');`

   `INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Jack Harknes', 38, '2008-06-08');`